### ISSUE TITLE
"github.copilot.inlineSuggest.enable" warning : "Deprecated: Please use github.copilot.editor.enableAutoCompletions instead.(2)"

### ISSUE THREAD

**Issue by MarArMar** [jamesonknutson/disable-copilot-comment-completions#9]
I got this warning in VSCode settings : 
"github.copilot.inlineSuggest.enable" warning : "Deprecated: Please use github.copilot.editor.enableAutoCompletions instead.(2)"

I believe this is the setting used by your extension ?

**Comment by jamesonknutson** [comment]
Yes, the extension needs to be updated to use the new setting value. Haven't gotten around to it yet, but the used setting (while deprecated, albeit) still works for the time being. You're welcome to submit a PR updating the setting key used if you have the time, otherwise I will get to this next time I'm working on updating the extension.

**Comment by JonathanMEdwards** [comment]
This extension dynamically toggles that setting in my settings.json file, which I am tracking in git, so that causes repeated changes.

**Comment by jamesonknutson** [comment]
This problem should now be fixed in the [latest release](https://github.com/jamesonknutson/disable-copilot-comment-completions/blob/master/CHANGELOG.md#201), available on the VSCode marketplace now.

> Updated the extension to use Copilot's new github.copilot.editor.enableAutoCompletions setting, instead of github.copilot.inlineSuggest.enable (which has been deprecated). This has the added bonus side effect of allowing you to manually trigger inline suggestions via VSCode's Alt + \ keybinding (command ID: editor.action.inlineSuggest.trigger). Fixes https://github.com/jamesonknutson/disable-copilot-comment-completions/issues/12, and https://github.com/jamesonknutson/disable-copilot-comment-completions/issues/9

If you guys find differently, let me know and I will revisit the problem.

Thanks for using my extension :)

@JonathanMEdwards @MarArMar
